packages/syntax/src/index.ts

```diff
@@ -1,5 +1,4 @@
 import Syntax from './Syntax';
-
-export { Variant, Language, SyntaxProps } from './types';
+export { Mode, Language, SyntaxProps } from './types';
```

Comment:
```suggestion
export { Language, SyntaxProps } from './types';
```

Reply:
I use the import in Code as well

Reply:
shouldn't Code define its own enum? If the enum and type is going to be shared between different components then it would make more sense for it to be exported by leafygreen provider.

Reply:
Fair. I can move this to Code locally